### ISSUE TITLE
Loading Current Controller not updating

### ISSUE THREAD

**Issue by mkelowna** [rechsteiner/Parchment#77]
I am using the following code to try and select a view controller - but its data structure is empty in

 
     
        let controllers = viewcontrollers(from: pagingItems)
        pagingViewController =  FixedPagingViewController(viewControllers: controllers, options:    CustomOptions())
       addChildViewController(pagingViewController!)
        viewHolder.addSubview((pagingViewController?.view)!)
        viewHolder.constrainToEdges((pagingViewController?.view)!)
       
        
        // Set our custom data source
        pagingViewController!.itemDelegate = self
        
        // Set the current date as the selected paging item
         let items = pagingViewController.items 
            let index = Defaults[.vcIndex]
            if index < items.count {
                pagingViewController!.selectPagingItem(pagingViewController.items [index])
            }
         pagingViewController?.didMove(toParentViewController: self)

In PagingViewController.swift, dataStructure is empty.
```
     open func selectPagingItem(_ pagingItem: T, animated: Bool = false) { 
       if let stateMachine = stateMachine {
      if let indexPath = dataStructure.indexPathForPagingItem(pagingItem) {
        let direction = dataStructure.directionForIndexPath(indexPath, currentPagingItem: pagingItem)
        stateMachine.fire(.select(
          pagingItem: pagingItem,
          direction: direction,
          animated: animated))
      }

```

**Comment by fatosdagdevir** [comment]
I found a temporary solution for this problem. 
It works in viewDidAppear.

 override func viewDidAppear(_ animated: Bool) {     pagingViewController.selectPagingItem(pagingViewController.items[2], animated: true) 
    }

**Comment by rechsteiner** [comment]
Looks like there was a bug when selecting items before the view had appeared. Should be fixed in #79. Thanks for catching that!
